g_docs.go

```diff
@@ -654,6 +628,33 @@ func parserComments(comments *ast.CommentGroup, funcName, controllerName, pkgpat
 	return nil
 }
 
+func consumes(a string) []string {
```

Comment:
I know it is `a` from above, but since you refactored into own function, should we rename this to something proper?

Reply:
hehe yeah, I tried changing it to accept earlier, but since the function was small-ish, changed it back to `a`, should we just use `accept`? 🤔

Reply:
Yes sure sound good!

Reply:
Addressed in https://github.com/zalora/bee/pull/37/commits/1adbbbcb39be40abedf8ba2361e7db9e225c9cf5